C/cp/BoyerMooreVote.c

```diff
@@ -0,0 +1,83 @@
+#include<stdio.h>
```

Comment:
Add a description/pitch about the Algorithm that you have implemented

Reply:
Not done yet. Kindly take a look at other Code Files to better understand the requirement.